cluster/gce/gci/configure-helper.sh

```diff
@@ -2090,9 +2110,13 @@ function start-kube-controller-manager {
     params+=" --pv-recycler-pod-template-filepath-hostpath=$PV_RECYCLER_OVERRIDE_TEMPLATE"
   fi
   if [[ -n "${RUN_CONTROLLERS:-}" ]]; then
-    params+=" --controllers=${RUN_CONTROLLERS}"
+    # Trim the `RUN_CONTROLLERS` value. This field is quoted which is
+    # incompatible with the `convert-manifest-params` format.
+    params+=" --controllers=${RUN_CONTROLLERS//\'}"
   fi
-
+  params+=" --log-file=${LOG_PATH}"
+  params+=" --logtostderr=false"
+  params=$(convert-manifest-params "${params}")
```

Comment:
I think `$(...)` should be quoted. I'm not sure it matters in this specific case, but it's fragile without the quotes, especially since it includes newline characters.
```suggestion
  params="$(convert-manifest-params "${params}")"
```

Same applies elsewhere.